s3tests_boto3/functional/test_s3.py

```diff
@@ -14942,12 +14942,27 @@ def _verify_records(records, bucket_name, event_type, src_keys, record_type, exp
     return len(keys_found) == expected_count
 
 
+def _verify_record_field(records, bucket_name, event_type, object_key, record_type, field_name, expected_value):
```

Comment:
This has also been defined in https://github.com/ceph/s3-tests/pull/667. This PR will need to be modified once that one is merged.